packages/components/src/components/date-picker/date-picker-native.tsx

```diff
@@ -4,7 +4,25 @@ import { toMoment } from '@deriv/shared';
 import Icon from '../icon';
 import Text from '../text';
 
-const Native = ({
+interface NativePropsT {
+    value: string;
+    id: string;
+    name: string;
+    label: string;
+    placeholder: string;
+    max_date: moment.Moment;
+    min_date: moment.Moment;
+    display_format: string;
+    data_testid: string;
+    hint?: string;
+    error?: string;
+    disabled: boolean;
+    onSelect: (new_date: string) => void;
```

Comment:
```suggestion
    onSelect: (selected_date: string) => void;
```
let's name it the same as it's named in the callback declaration. callback is onSelectCalendarNative()🙂